data/hai/hai reveal 4 middle.txt

```diff
@@ -1391,7 +1397,37 @@ mission "Hai Reveal [A10] Demanding Satisfaction"
 			choice
 				`	(Walk to her.)`
 			`	"Captain <first> <last> led great fleets during a civil war of the humans. We will have the honor of fighting alongside this warrior of the humans."`
-			`	He looks at you appraisingly. "That is well, but there is a matter yet to be addressed. We cannot just 'waltz' through Hai and human space without being blasted. How will you convince your Elders to allow us passage?"`
+			branch nobody
+				not "Wanderers: Defend Vara Ke'sok: done"
+			`	"Indeed, we have had the honor of fighting against this warrior of humans. Quite a troublemaker on the battlefield." He looks at you. "Your professor friend was chatty before you left, and it's been easy to track you since then. We've been studying your actions for quite a while. Why are you here now, as an ally of this True Hai?"`
+			choice
+				`	"You and I have crossed swords before, but this is a matter that concerns us both: the safety and honor of the Hai."`
+					goto honor
+				`	"I fight for money and prestige. The Hai are paying plenty of both for protection against a pirate threat."`
+					goto mercenary
+				`	"I assumed furballs like you were too brain-dead to count to two."`
+			`	Teeneep rolls her eyes and makes no attempt to intervene as General Choot'k moves rapidly to cut your head in twain.`
+				die
+			label honor
+			`	He stares intensely at you with a piercing gaze. After a long moment, General Choot'k says, "You have proven an honorable warrior, Captain <last>, even if you have chosen the wrong side of one war. Yes, I am willing to fight alongside you, defending the Hai.`
```

Comment:
```suggestion
			`	He stares intensely at you with a piercing gaze. After a long moment, General Choot'k says, "You have proven an honorable warrior, Captain <last>, even if you have chosen the wrong side of one war. Yes, I am willing to fight alongside you, defending the Hai.”`
```

Reply:
No because:

1. Characters in data files must be printable 7-bit ascii (`"` not `”`)
2. Only the last paragraph in a quote is terminated by a `"`

Reply:
1. I am aware, for some reason my computer doesn’t use ASCII. I have attempted to change it to normal. 
2. In that case, this can be resolved.

Reply:
> I am aware, for some reason my computer doesn’t use ASCII. I have attempted to change it to normal.

Many programs will automatically change `"` and `'` to the beginning or ending curly quotes. You may have to disable that in your browser settings.

Reply:
Thanks for the information!

Reply:
This can be resolved.